<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addquestionwindow</class>
 <widget class="QWidget" name="addquestionwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление вопросов</string>
  </property>
  <widget class="QPushButton" name="prevQuestion_pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Предыдущий вопрос</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="questionText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>681</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавление вопросов:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Номер вопроса:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="questionNumber">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Текст вопроса:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>681</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ответы на вопросы, ответы должны быть разделены знаком &amp;quot;;&amp;quot;, а правильный ответ должен быть&lt;/p&gt;&lt;p&gt; в круглых скобках. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="testAnswers">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>681</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="nextQuestion_pushButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>450</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Следующий вопрос</string>
   </property>
  </widget>
  <widget class="QPushButton" name="finishAdding_pushButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>450</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Закончить добавление вопросов</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
